front/src/Api/IframeListener.ts

```diff
@@ -328,7 +319,7 @@ class IframeListener {
     }
 
     registerScript(scriptUrl: string, enableModuleMode: boolean = true): Promise<void> {
-        return new Promise<void>((resolve, reject) => {
+        return new Promise<void>((resolve) => {
```

Comment:
We should probably put a reject here?

Reply:
In fact no, because onerror is not triggered across domains (see https://www.w3.org/TR/2011/WD-html5-20110525/the-iframe-element.html#the-iframe-element)